src/components/ui/common/Source.js

```diff
@@ -35,7 +36,5 @@ exports.getFileTitle = function(expectedTitle) {
 };
 
 exports.getLoadingMessage = function() {
-    return element(by.css('.noPreview .empty')).getText().then(function(text) {
-        expect(text).toEqual('Generating Blueprint Source View...');
-    });
+    expect(browser.driver.findElement(by.css('.noPreview p')).getText()).toBe('Generating Blueprint Source View...');
```

Comment:
It's a trick to make the check before the source is loaded. http://stackoverflow.com/a/30137524